notebooks/02_model/ncf_deep_dive.ipynb

```diff
@@ -30,13 +30,31 @@
    "execution_count": 1,
```

Comment:
<p>would it be possible to erase the output of this cell? it just shows a warning which is not very valuable for a reader</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/microsoft/recommenders/pull/1092/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='36'/>